staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -41,7 +41,7 @@ const (
 
 // FailurePolicyType specifies a failure policy that defines how unrecognized errors from the admission endpoint are handled.
 // +enum
-type FailurePolicyType string
+type FailurePolicyType = v1.FailurePolicyType
```

Comment:
@cici37 @liggitt Is this safe? I can't remember where we landed on the use of aliases between versions.

Reply:
FYI We did the similar type alias for ScopeType and Rule. I proposed it mainly to avoid unnecessary conversion since FailurePolicy is one of the key staff Validate really cares about.

Reply:
we've successfully used them in admissionregistration and meta/v1beta1 definitions without any apparent issues